.travis.yml

```diff
@@ -2,4 +2,4 @@ language: node_js
 node_js:
```

Comment:
you might want to run this on a lot more node versions; and might want the linter to run separately from tests - if so, you can follow this pattern: https://github.com/ljharb/unbox-primitive/blob/master/.travis.yml#L1-L7

the only hitch is you'd need to define `"test"` as `"npm run tests-only"`, and put the actual test command in `"tests-only"`.

Reply:
Nice repository!